content/docs/command-reference/plots/index.md

```diff
@@ -338,116 +38,3 @@ this feature to render DVC plots without an Internet connection, below.
 - `-q`, `--quiet` - do not write anything to standard output.
 
 - `-v`, `--verbose` - displays detailed tracing information.
-
-## Example: Offline HTML Template
-
-The plots generated by `dvc plots` uses Vega-Lite JavaScript libraries, and by
-default these load [online resources](https://vega.github.io/vega/usage/#embed).
-There may be times when you need to produce plots without Internet access, or
```

Comment:
I think all these examples still belong more to the cmf ref, at least for consistency (open to discuss) ...

Reply:
> Would you suggest putting them in https://dvc.org/doc/command-reference/plots/show?

No @dberenbaum I was thinking to keep them where they were but I realize now that's out of context since all the plot template info is in the new guide now. But...